docs/cli/usage/permission-boundary.md

```diff
@@ -0,0 +1,16 @@
+---
+title: IAM Role Permission Boundary
+description: Apply a Permission Boundary to all IAM Roles created by Amplify CLI.
+---
+
+To set the maximum permissions that can be granted to IAM Roles created by Amplify CLI, a [Permission Boundary](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html) can be configured for the project.
```

Comment:
Could we include a bit of background regarding what Amplify does behind the scenes with respect to creating IAM roles? To the extent that we document that elsewhere, let's link to that resource.